usaspending_api/search/models/transaction_search.py

```diff
@@ -5,7 +5,7 @@
 
 class AbstractTransactionSearch(models.Model):
     transaction = models.OneToOneField("awards.TransactionNormalized", on_delete=models.DO_NOTHING, primary_key=True)
-    award = models.ForeignKey("awards.Award", on_delete=models.DO_NOTHING, null=True)
+    award = models.ForeignKey("awards.Award", on_delete=models.DO_NOTHING, null=True, related_name="transactions")
```

Comment:
Will need to override this in the `TransactionSearchDownloadView` with something such as:
```
award = models.ForeignKey("awards.Award", on_delete=models.DO_NOTHING, null=True, related_name="transactions")
```
because there cannot be two fields with the same `related_name`.